cobra/io/sbml.py

```diff
@@ -1057,14 +1052,20 @@ def _model_to_sbml(cobra_model, f_replace=None, units=True):
             if f_replace and F_GENE_REV in f_replace:
                 gpr = gpr.replace('(', '( ')
                 gpr = gpr.replace(')', ' )')
+                gpr = gpr.replace('  ', ' ')
```

Comment:
Since the next line anyway splits one a single space I suggest to drop this line and rather replace the next one with splitting on any whitespace:
```py
tokens = gpr.split()
```

Reply:
Did not know about that, changed.